selfdrive/test/process_replay/test_fuzzy.py

```diff
@@ -1,174 +1,26 @@
-    'source': st.just(8),  # BMX055
-    'gyroUncalibrated': r['SensorEventData.SensorVec'],
-  })
-  r['SensorEventData_accel'] = st.fixed_dictionaries({
-    'version': st.just(1),
-    'sensor': st.just(1),
-    'type': st.just(1),
-    'timestamp': st.integers(min_value=0, max_value=2**63 - 1),
-    'source': st.just(8),  # BMX055
-    'acceleration': r['SensorEventData.SensorVec'],
-  })
-  r['SensorEvents'] = st.lists(st.one_of(r['SensorEventData_gyro'], r['SensorEventData_accel']), min_size=1)
-  r['GpsLocationExternal'] = st.fixed_dictionaries({
-    'flags': st.just(1),
-    'latitude': floats(),
-    'longitude': floats(),
-    'altitude': floats(),
-    'speed': floats(width=32),
-    'bearingDeg': floats(width=32),
-    'accuracy': floats(width=32),
-    'timestamp': st.integers(min_value=0, max_value=2**63 - 1),
-    'source': st.just(6),  # Ublox
-    'vNED': st.lists(floats(width=32), min_size=3, max_size=3),
-    'verticalAccuracy': floats(width=32),
-    'bearingAccuracyDeg': floats(width=32),
-    'speedAccuracy': floats(width=32),
-  })
-  r['LiveCalibration'] = st.fixed_dictionaries({
-    'rpyCalib': st.lists(floats(width=32), min_size=3, max_size=3),
-  })
-
-  return st.lists(st.one_of(*[get_event_union_strategy(r, n) for n in event_types]))
-
-
-def get_strategy_for_process(process, finite=False):
-  return get_strategy_for_events(get_process_config(process).pubs, finite)
-
-
-def convert_to_lr(msgs):
-  return [log.Event.new_message(**m).as_reader() for m in msgs]
-
-
-def is_finite(d, exclude=[], prefix=""):  # pylint: disable=dangerous-default-value
-  ret = True
-  for k, v in d.items():
-    name = prefix + f"{k}"
-    if name in exclude:
-      continue
-
-    if isinstance(v, dict):
-      if not is_finite(v, exclude, name + "."):
-        ret = False
-    else:
-      try:
-        if not np.isfinite(v).all():
-          note((name, v))
-          ret = False
-      except TypeError:
-        pass
-
-  return ret
-
-
-def test_process(dat, name):
-  cfg = get_process_config(name)
-  lr = convert_to_lr(dat)
-  pr.TIMEOUT = 0.1
-  return pr.replay_process(cfg, lr, TOYOTA.COROLLA_TSS2)
-
-
-class TestFuzzy(unittest.TestCase):
-  @given(get_strategy_for_process('paramsd'))
-  @settings(deadline=1000)
-  def test_paramsd(self, dat):
-    for r in test_process(dat, 'paramsd'):
-      d = r.liveParameters.to_dict()
-      assert is_finite(d)
-
-  @given(get_strategy_for_process('locationd', finite=True))
-  @settings(deadline=1000)
-  def test_locationd(self, dat):
-    exclude = [
-      'positionGeodetic.std',
-      'velocityNED.std',
-      'orientationNED.std',
-      'calibratedOrientationECEF.std',
-    ]
-    for r in test_process(dat, 'locationd'):
-      d = r.liveLocationKalman.to_dict()
-      assert is_finite(d, exclude)
+class TestFuzzProcesses(unittest.TestCase):
 
+  @parameterized.expand([(cfg.proc_name, cfg) for cfg in pr.CONFIGS if cfg.proc_name not in NOT_TESTED])
```

Comment:
maybe move the list comprehension to the top? there's a lot going on in this line